src/PrintOrder/PrintOrder.js

```diff
@@ -35,7 +41,9 @@ export const PrintOrderComponent = ({ mutator, order, orderLine, onCancel }) =>
         ? { lines: await getExportData(mutator, linesToPrint, [order], intl) }
         : await getOrderPrintData(ky, order);
 
-      setPrintableOrder(hydrateOrderToPrint({
+      setPrintableOrder(await hydrateOrderToPrint({
+        ky,
```

Comment:
don't put ky to hydrateOrderToPrint, introduce hook that will return price based on order lines amount

Reply:
Sure. Let me think about it.

Reply:
updated